packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -114,7 +71,8 @@ class HashUrlStrategy extends UrlStrategy {
   final PlatformLocation _platformLocation;
 
   @override
-  ui.VoidCallback addPopStateListener(html.EventListener fn) {
+  ui.VoidCallback addPopStateListener(EventListener fn) {
+    assert(fn is html.EventListener);
```

Comment:
No need to assert anything here? I'd say maybe `!= null`, but it's not an `EventListener?` so it can't be null either. All the checks are going to be made for free by the language at this point?

Reply:
Hahaha beat me to it xD

Reply:
`EventListener` isn't the same type as `html.EventListener`, that's why I added the type check. But now that I've seen the warning and thought about it, `EventListener` is more generic and accepts any argument, so of course it can be used in lieu of `html.EventListener` with no problems.

I removed the type check.